pkg/sources/github/github.go

```diff
@@ -577,11 +646,12 @@ func (s *Source) scan(ctx context.Context, installationClient *github.Client, ch
 	}
 
 	_ = s.jobPool.Wait()
-	if len(scanErrs) == 0 {
-		s.SetProgressComplete(len(s.repos), len(s.repos), "Completed Github scan", "")
+	if scanErrs.Count() > 0 {
+		s.log.V(2).Info("encountered errors while scanning", "count", scanErrs.Count(), "errors", scanErrs)
 	}
+	s.SetProgressComplete(len(s.repos), len(s.repos), "Completed Github scan", "")
```

Comment:
was it intentional to change the progress completion behavior of this function? before, completion would only be set if there were no errors, but now completion is set every time.

Reply:
Yea, I updated it to match all our other sources. There is usually a chance there is at least 1 error during the scan. If we don't set the progress to complete the FE will interpret it as 'the scan is still running". We've had discussions about this in the past and resorted to the less strict approach. Ideally we have maybe some threshold for errors, that when it gets crossed we consider the source in an unhealthy state. We currently just log most errors and complete the scan if possible.